src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -0,0 +1,83 @@
+
+## Preliminary steps
+
+Migrations will create new changesets in each of the repositories. Before starting, it’s highly recommended to review all pending sign-offs in Elmo for the current Firefox Beta. More details about the sign-off process are available in [this document](../review/signoffs.md).
+
+## Stop sync in Pontoon
+
+Before starting the migration process, sync needs to be suspended for all projects relying on the l10n-central repositories. The list of affected projects is:
+* Firefox
+* Thunderbird
+* Seamonkey
+* Lightning
+
+In order to disable sync, access each project from the [admin panel](https://pontoon.mozilla.org/admin/), and select the checkbox `SYNC DISABLED` at the bottom of the page, then `SAVE PROJECT`.
+
+This is needed for several reasons:
+* Pontoon takes a lot of time processing this amount of changes, creating a bottleneck in the sync for all projects.
+* The migrated strings need to be added in the same cycle as the new strings for the source language (`en-US`). If they’re added before, Pontoon will ignore them, creating a misalignment between internal database and l10n repositories. If they’re added after, these strings will be displayed as missing in Pontoon, and some locales might try to translate them while it’s not needed.
+
+The last step here is to make sure that the current sync process has completed, then increase the resources assigned to the Heroku worker:
+* Check the [Sync Logs](https://pontoon.mozilla.org/sync/log/) page, click on the last one, and make sure that there are no projects with `---` in the `DURATION` field.
+* In [Heroku](https://dashboard.heroku.com/apps/mozilla-pontoon/resources), switch the *worker* from `PM` (Performance-M dynos) to `PL` (Performance-L dynos).
+
+It’s important to make sure that there is no sync in progress when upgrading the worker, because that will kill any pending process.
+
+## Push updates to gecko-strings
+
+Assuming the new strings are available in the [quarantine repository](https://hg.mozilla.org/users/axel_mozilla.com/gecko-strings-quarantine), it’s now time to push the updates to the official [gecko-strings repository](https://hg.mozilla.org/l10n/gecko-strings).
+
+Pontoon relies on a [different repository](https://hg.mozilla.org/users/m_owca.info/firefox-central/), that is generated every 20 minutes based on the content of `gecko-strings`. That’s why it’s useful to run this update before starting the actual migrations.
+
+## Run migrations
+
+A series of tools and helpers to run migrations is available in [this repository](https://github.com/flodolo/fluent-migrations):
+* Clone the repository locally and follow the instructions available in the README to set the config.
+* Add the new migration recipe in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder.
```

Comment:
Filed https://github.com/flodolo/fluent-migrations/issues/3 to make the folder structure a bit more clear. I think that belongs more into the realm of that repo?

Reply:
Yes, I think so too.